apps/modernization-ui/src/apps/page-builder/pages/EditPage/ManageTabs/ManageTabs.tsx

```diff
@@ -1,28 +1,60 @@
-import React from 'react';
-import { PagesTab, Tab } from 'apps/page-builder/generated';
+// import { useState } from 'react';
+import { PagesTab } from 'apps/page-builder/generated';
 import './ManageTabs.scss';
 import { Icon } from '@trussworks/react-uswds';
 import { Icon as IconComponent } from 'components/Icon/Icon';
 
 type Props = {
-    tabs: Tab[];
+    tabs: PagesTab[];
     setSelectedEditTab: (tab: PagesTab, index: number) => void;
+    setDeleteTab: (tab: PagesTab) => void;
+    selectedForDelete: PagesTab | undefined;
+    setSelectedForDelete: (tab: PagesTab | undefined) => void;
+    reset: () => void;
 };
 
-const ManageTabs = ({ tabs, setSelectedEditTab }: Props) => {
+const ManageTabs = ({
+    tabs,
+    setSelectedEditTab,
+    setDeleteTab,
+    selectedForDelete,
+    setSelectedForDelete,
+    reset
+}: Props) => {
     return (
         <div className="manage-tabs">
             {tabs.map((tab, i) => {
                 return (
-                    <div key={i} className="manage-tabs__tile" style={{ listStyle: 'none' }}>
-                        <div className="manage-tabs__label">
+                    <div
+                        key={i}
+                        className={`manage-tabs__tile ${tab.id === selectedForDelete?.id ? 'delete' : ''}`}
+                        style={{ listStyle: 'none' }}>
+                        <div className="manage-tabs__handle">
                             <IconComponent name="drag" />
+                            <p>Delete this tab?</p>
+                        </div>
+                        <div className="manage-tabs__label">
                             <IconComponent name="folder" />
-                            {tab.name}
+                            {tab.name} {tab.sections && tab.sections.length ? '(' + tab.sections!.length + ')' : '(0)'}
```

Comment:
Suggestion: i think this can be simplified. (untested)

```typescript
 {`(${tab.sections?.length ?? 0})`}
```

Reply:
very nice! 💎